### ISSUE TITLE
Demo deployment doesn't work

### ISSUE THREAD

**Issue by mm0** [aalaesar/install_nextcloud#27]
Looks like this is because of the following error: 

```
changed: [] => {
    "changed": true,
    "cmd": [
        "php",
        "occ",
        "maintenance:install",
        "--database=mysql",
        "--database-host=127.0.0.1",
        "--database-name=nextcloud",
        "--database-user=ncadmin",
        "--database-pass=asd",
        "--admin-user=admin",
        "--admin-pass=asd",
        "--data-dir=/var/ncdata"
    ],
    "delta": "0:00:00.021375",
    "end": "2017-03-07 06:41:29.485354",
    "invocation": {
        "module_args": {
            "_raw_params": "php occ maintenance:install --database=mysql --database-host=127.0.0.1 --database-name=nextcloud --database-user=ncadmin --database-pass=asd --admin-user=admin --admin-pass=asd --data-dir=/var/ncdata",
            "_uses_shell": false,
            "chdir": "/opt/nextcloud",
            "creates": "/opt/nextcloud/config/config.php",
            "executable": null,
            "removes": null,
            "warn": true
        },
        "module_name": "command"
    },
    "rc": 0,
    "start": "2017-03-07 06:41:29.463979",
    "stderr": "",
    "stdout": "This version of Nextcloud requires at least PHP 5.6.0\nYou are currently running 5.5.9-1ubuntu4.21. Please update your PHP version.",
    "stdout_lines": [
        "This version of Nextcloud requires at least PHP 5.6.0",
        "You are currently running 5.5.9-1ubuntu4.21. Please update your PHP version."
    ],
    "warnings": []
}
```

**Comment by aalaesar** [comment]
Hello @mm0. Looks like an unmet dependency version for nextcloud.
I'm guessing you're running the role on a target on Ubuntu trusty 14.04
If you' re using the demo case, the role do an installation of the latest version available : 11.0.2 at the moment
However, nc 11+ requires php 5.6 as a minimum while trusty doesn't have 5.5+ available in its standards & backports repo.
So unless you add a custom repository with a newer php (which this role is not intended to fix) you won't be able to install nextcloud 11.
You still can install the latest v10 on trusty which is still a supported release.
You just have to add 
```YAML
nextcloud_version_major: 10
```
To your playbook.
This is what is used in the tests/test.yml playbook for Travis CI.
Following this issue, I'll add a assert or warning about this case in the readme.
Regards, Aal.

**Comment by mm0** [comment]
Got it. Thanks. I ended up going with xenial
